sdk/template/template/package.json

```diff
@@ -97,22 +97,19 @@
     "@azure/identity": "^4.0.1",
     "@azure-tools/test-utils": "^1.0.0",
     "@microsoft/api-extractor": "^7.40.3",
-    "@types/fs-extra": "^8.0.0",
     "@types/node": "^18.0.0",
-    "@vitest/browser": "^1.3.1",
-    "@vitest/coverage-istanbul": "^1.3.1",
+    "@vitest/browser": "^1.5.3",
+    "@vitest/coverage-istanbul": "^1.5.3",
     "cross-env": "^7.0.2",
     "dotenv": "^16.0.0",
     "eslint": "^8.56.0",
-    "inherits": "^2.0.3",
+    "loupe": "^3.1.0",
```

Comment:
what the heck do we use this for?

Reply:
It's because there's a bug in vitest: https://github.com/vitest-dev/vitest/issues/5332